test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuccessfulAuthenticatedAction.scala

```diff
@@ -80,6 +80,14 @@ class SuccessfulRequestActions(operator: Operator, c: Case = Cases.btiCaseExampl
     new SuccessfulAuthenticatedAction(operator),
     new ExistingCaseActionFactory(reference, c),
     new HaveRightPermissionsActionFactory
-  ) {
+  ) {}
 
-}
\ No newline at end of file
+
+
+class RequestActionsWithPermissions(permissions : Set[Permission], reference: String = "test-reference",  c: Case = Cases.btiCaseExample)
+  extends RequestActions(
+    new CheckPermissionsAction,
+    new SuccessfulAuthenticatedAction(permissions = permissions ++ Set(Permission.VIEW_CASES)),
```

Comment:
What does the `VIEW_CASES` permission do?

Reply:
Doesnt everyone have the permission to view cases?

Reply:
I think is the idea that we will have this weird SysAdmin which does things more complicated